app/helpers/time-ago.js

```diff
@@ -0,0 +1,8 @@
+import { helper } from '@ember/component/helper';
+import moment from 'moment';
```

Comment:
`moment` is quite heavy, if you want to reduce bundle size, try [`dayjs`](https://github.com/iamkun/dayjs/blob/master/docs/en/Plugin.md#relativetime)

Reply:
Cant find support for time ago in `dayjs`. is there any?

Reply:
https://github.com/iamkun/dayjs/blob/master/docs/en/Plugin.md#relativetime

^ The link I gave before, time ago = relative time

Reply:
@blazeu Nice, I'll have a look

Reply:
Not gonna use it?

Reply:
@blazeu Next pull request, I'm afraid there is a gotcha. And take another day to fix.